etc/pip-requires-client

```diff
@@ -1,18 +1,8 @@
 # All dependencies needed to run rucio client should be defined here
 setuptools>=36.8.0,<45.0.0; python_version < '3.0'          # Python packaging utilities (36.8.0 last py2.6 compatible version, 45.0 only Py3 compatible)
 setuptools>=45.0.0,<=50.3.2; python_version >= '3.0'        # Python packaging utilities
-argcomplete>=1.9.0,<=1.12.2                                 # Bash tab completion for argparse
 requests>=2.20.0,<=2.25.0                                   # Python HTTP for Humans.
 urllib3>=1.24.2,<1.26                                       # HTTP library with thread-safe connection pooling, file post, etc.
 dogpile.cache>=0.6.5,<0.7.0; python_version < '3.0'         # Caching API plugins (0.7.0 only Py3 compatible)
 dogpile.cache>=0.6.5,<=1.1.1; python_version >= '3.0'       # Caching API plugins
-boto>=2.49.0,<2.50.0                                        # S3 boto protocol
-tabulate>=0.8.0,<0.9.0                                      # Pretty-print tabular data
-progressbar2>=3.39.0,<=3.53.1                               # Text progress bar
-bz2file>=0.98,<0.99                                         # Read and write bzip2-compressed files.
-python-magic>=0.4.15,<0.5.0                                 # File type identification using libmagic
-futures>=3.2.0; python_version < '3.0'                      # Clean single-source support for Python 3 and 2
 six>=1.12.0<1.16.0                                          # Python 2 and 3 compatibility utilities
-boto3>=1.9.130,<1.17.0                                      # S3 boto protocol (new version)
-pysftp>=0.2.9,<0.3                                          # Used by sftp protocol
-jsonschema>=3.2.0                                           # For JSON schema validation (Policy modules)
```

Comment:
Is this not used anywhere?

Reply:
boto3 is only used by the server to sign URLs. I don't think it was ever needed on the client side. pysftp is only needed for the sftp protocol so I made it optional. jsonschema seemed to be only used in the policy packages which I don't think are needed for the client. If I'm wrong and they are needed, this dependency should be kept.